hooks/pager.ts

```diff
@@ -35,7 +35,9 @@ export function usePager<T extends AidboxResource>(
         resources,
         {
             loadNext: () => setPageToLoad((currentPage) => currentPage + 1),
-            hasNext: isSuccess(resources) && !!_.find(resources.data.link, { relation: 'next' }),
+            hasNext:
+                isSuccess(resources) &&
+                Boolean(resources.data.link && resources.data.link.some((link) => link.relation == 'next')),
```

Comment:
We usually use === and !== by default for all conditions (it makes life easier because you don't need to think about how it works)